public/app/features/scenes/components/VizPanel/VizPanel.tsx

```diff
@@ -78,6 +81,15 @@ export class VizPanel<TOptions = {}, TFieldConfig = {}> extends SceneObjectBase<
     });
 
     this._plugin = plugin;
+
+    sceneGraph.getData(this).subscribeToState({
+      next: (data) => {
+        this.setState({
+          timeOverrideInfo: this.getTimeOverrideInfo(data.timeFrom, data.timeShift),
+        });
+      },
+    });
```

Comment:
this should not be needed, the timeshift info can be extracted from the PanelData (data.request.timeInfo)